_posts/2020-03-19-quarkus-1-3-0-final-released.adoc

```diff
@@ -0,0 +1,206 @@
+either using Jib, Docker or S2I.
+
+More information in the https://quarkus.io/guides/container-image[Container Image guide].
+
+==== OptaPlanner
+
+OptaPlanner is an AI constraint solver.
+It optimizes planning and scheduling problems.
+
+The OptaPlanner Quarkus extension brings all the features of OptaPlanner to Quarkus,
+even in native executables.
+If you want to discover how OptaPlanner can help, please read https://quarkus.io/guides/optaplanner[the OptaPlanner guide].
+
+==== Security JPA
+
+Want your users in a database managed by Hibernate ORM?
+Just add a couple of annotations and https://quarkus.io/guides/security-jpa[the Security JPA extension] gets you covered.
+
+==== Spring Cloud Config Client
+
+We extended our Spring compatibility layer with a Spring Cloud Config Client.
+
+You can now https://quarkus.io/guides/spring-cloud-config-client[get your configuration from a Spring Cloud Config server].
+
+==== AWS X-Ray
+
+Support for AWS X-Ray was added so you can use it with your AWS lambdas, in JVM mode and native mode.
+
+==== Qpid JMS
+
+We now recommend the usage of the Qpid JMS extension instead of the Artemis JMS one.
+Features are the same, it just adds more flexibility.
+
+The Artemis JMS extension is still there.
+
+More information in https://quarkus.io/guides/jms[the JMS guide].
+
+==== Liquibase
+
+We had a Flyway extension for a while, you can now also use Liquibase to handle your database schema migrations.
+
+More on this in https://quarkus.io/guides/liquibase[the Liquibase guide].
+
+==== Reactive MongoDB with Panache
+
+Love reactive?
+Love MongoDB?
+Love simple code?
+
+Here comes the Reactive MongoDB with Panache extension.
+More details in https://quarkus.io/guides/mongodb-panache[the MongoDB with Panache documentation].
+
+=== GraalVM 20.0 support
+
+1.3.0.Final supports four flavors of GraalVM:
+
+ * GraalVM 19.3.1 - JDK 8
+ * GraalVM 19.3.1 - JDK 11
+ * GraalVM 20.0.0 - JDK 8
+ * GraalVM 20.0.0 - JDK 11
+
+[NOTE]
+====
+To summarize, for Quarkus, you can use:
+
+ * JDK 8, 11, 12, 13 or 14 in JVM mode
+ * JDK 8 or 11 with GraalVM 19.3.1 or 20.0.0 to build native executables
+====
+
+[TIP]
+====
+JDK 11 support in GraalVM is still in tech preview.
+On our side, we did not encounter any specific issues except for the compilation to be ~20% slower.
+====
+
+=== Hibernate ORM
+
+Automatic association management via bytecode enhancement is disabled from now on.
+
+It is possible to disable the second level caching
+by setting the `quarkus.hibernate-orm.second-level-caching-enabled` configuration property to `false`.
+
+=== Spring compatibility layer
+
+The Spring compatibility layer is now based on trimmed down artifacts only including the necessary APIs and annotations instead of the full Spring jars.
+
+Less dependencies, less code and you only have the supported features around.
+
+=== And more...
+
+ * Several improvements in our Gradle and Kotlin support
+ * Kogito has been updated to 0.8
```

Comment:
Are we sure about this one? I thought `0.8` was only on `master`...

Reply:
Ah yes, right.